api/src/functions/express/app.ts

```diff
@@ -209,8 +209,9 @@ const dynamicsHousingRegistrationStatusClient = DynamicsHousingRegistrationStatu
   housingPropertyInterestClient: dynamicsHousingPropertyInterestClient,
 });
 
-const BUSINESS_NAME_BASE_URL =
-  process.env.BUSINESS_NAME_BASE_URL || `http://${IS_DOCKER ? "wiremock" : "localhost"}:9000`;
+const BUSINESS_NAME_BASE_URL = process.env.USE_WIREMOCK_FOR_FORMATION_AND_BUSINESS_SEARCH
+  ? "http://localhost:9000"
+  : process.env.BUSINESS_NAME_BASE_URL || `http://${IS_DOCKER ? "wiremock" : "localhost"}:9000`;
```

Comment:
[dust] Just musing here, I wonder if there's ever a situation where `USE_WIREMOCK_FOR_FORMATION_AND_BUSINESS_SEARCH` is false but we point to wiremock anyway because `IS_DOCKER` is true.

Totally fine with the code as written, just curious.

Reply:
if that scenario is true, wouldn't everything work as is, meaning the right hand side of the ternary is checked.